13/umbraco-cms/reference/webhooks/README.md

```diff
@@ -119,6 +119,56 @@ This is a list of all the current events that are available through Umbraco. If
 builder.WebhookEvents().Clear().AddCms(false);
 ```
 
+## Replace Webhook Events
+Sometimes it is desirable to modify one of the standard Umbraco webhooks, for example, to change the Payload. This can be done by adding a custom implementation, as shown in the code example below:
```

Comment:
```suggestion
## Replace Webhook Events

Sometimes it is desirable to modify one of the standard Umbraco webhooks, for example, to change the Payload. This can be done by adding a custom implementation, as shown in the code example below:
```